### DISCUSSION TITLE
Get latest added Id

### DISCUSSION THREAD

**Discussion by Annie-khalid** [supabase/supabase#815] [Questions] [answered]
how i can get latest added Id in tabel? 
I have two tables that's have foreign key relation

**Comment by kiwicopple** [comment, marked as answer]
Hi @Annie-khalid can you share with us what type of column the ID is? If you used a sequence, then you can get the current value:

https://stackoverflow.com/questions/14886048/options-to-retrieve-the-current-on-a-moment-of-running-query-sequence-value

**Reply by Annie-khalid** [reply]
Thanks...my problem is solved already.
